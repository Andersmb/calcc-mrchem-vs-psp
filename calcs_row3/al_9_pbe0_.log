Starting job 5688395 on c10-[11,17],c11-[20,35] at Fri May 20 10:46:24 CEST 2022

Copying al_9_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying al_9_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688395      al_9_pbe0_        180                                             00:32:48      0:0 
5688395.bat+      batch         48        1   00:00:02          0   00:00:02   00:32:48      0:0 
5688395.ext+     extern        180        4   00:00:00          3   00:00:00   00:32:48      0:0 
5688395.0      mrchem.x        180       30   00:32:20         29   01:18:33   00:32:31      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688395                                                                          
5688395.bat+     19220K          0     19220K        0              0          0 
5688395.ext+          0          3          0        0              3          0 
5688395.0      8249832K         29 3970787191        0              9          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688395                                                                                               
5688395.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688395.ext+        0.00M               3          0.00M            0                3              0 
5688395.0           7.32M               0          7.31M       67.01M               17         45.73M 

Job 5688395 completed at Fri May 20 11:19:13 CEST 2022
