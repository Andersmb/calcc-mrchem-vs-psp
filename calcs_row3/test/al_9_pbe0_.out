                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -1.138094
    1       O            :        0.000000        0.000000        1.919293
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              11
 Beta electrons          :                                              10
 Total electrons         :                                              21
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999997526041e-01
    1    1    a          :                              9.999999963316e-01
    2    1    a          :                              9.999999525847e-01
    3    1    a          :                              9.999999319871e-01
    4    1    a          :                              9.999999552741e-01
    5    1    a          :                              9.999999515595e-01
    6    1    a          :                              9.999995588025e-01
    7    1    a          :                              9.999996272352e-01
    8    1    a          :                              1.000002356297e+00
    9    1    a          :                              1.000002532469e+00
   10    1    a          :                              9.999991808385e-01
   11    1    b          :                              9.999996745002e-01
   12    1    b          :                              9.999999963323e-01
   13    1    b          :                              9.999999525847e-01
   14    1    b          :                              9.999999316745e-01
   15    1    b          :                              9.999999357427e-01
   16    1    b          :                              9.999999556968e-01
   17    1    b          :                              9.999994953078e-01
   18    1    b          :                              9.999996368178e-01
   19    1    b          :                              1.000002330324e+00
   20    1    b          :                              1.000002532523e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                316.180959264676
 E-N energy              :            (au)               -825.709496773230
 Coulomb energy          :            (au)                187.537203361017
 Exchange energy         :            (au)                 -6.691633242251
 X-C energy              :            (au)                -20.715508360631
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 34.015967513809
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -349.398475750418
 Nuclear energy          :            (au)                 34.015967513809
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.153825082366e+02
                         :      (kcal/mol)             -1.979055102754e+05
                         :        (kJ/mol)             -8.280366555299e+05
                         :            (eV)             -8.581994869661e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -55.928912166535
    1    1    a          :            (au)                -18.249373388015
    2    1    a          :            (au)                 -4.290434125338
    3    1    a          :            (au)                 -2.817720515544
    4    1    a          :            (au)                 -2.817432775865
    5    1    a          :            (au)                 -2.809402383327
    6    1    a          :            (au)                 -0.537343058779
    7    1    a          :            (au)                 -0.341793970644
    8    1    a          :            (au)                  0.102504361059
    9    1    a          :            (au)                  0.103898427178
   10    1    a          :            (au)                  0.116599502434
   11    1    b          :            (au)                -55.924752008243
   12    1    b          :            (au)                -18.246697897851
   13    1    b          :            (au)                 -4.282205512008
   14    1    b          :            (au)                 -2.812419762377
   15    1    b          :            (au)                 -2.812172350359
   16    1    b          :            (au)                 -2.802043392041
   17    1    b          :            (au)                 -0.503120362767
   18    1    b          :            (au)                  0.061197220324
   19    1    b          :            (au)                  0.107859804721
   20    1    b          :            (au)                  0.120751104502
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -174.563013249476
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.582576e+00        -315.382508236609         -3.153825e+02
    1          1.308605e+00        -316.170340788188         -7.878326e-01
    2          9.275602e-01        -317.327471762656         -1.157131e+00
    3          3.365922e-01        -317.418157136678         -9.068537e-02
    4          2.068303e-01        -317.447085609029         -2.892847e-02
    5          1.424037e-01        -317.462538765195         -1.545316e-02
    6          3.668780e-02        -317.464146342405         -1.607577e-03
    7          2.952655e-02        -317.465086799648         -9.404572e-04
    8          3.079479e-02        -317.465690245491         -6.034458e-04
    9          3.700244e-02        -317.466187875662         -4.976302e-04
   10          3.184268e-02        -317.466531393400         -3.435177e-04
   11          3.258034e-02        -317.466789068559         -2.576752e-04
   12          2.268318e-02        -317.466909784113         -1.207156e-04
   13          1.107655e-02        -317.466947847880         -3.806377e-05
   14          4.744460e-03        -317.466961305948         -1.345807e-05
   15          4.141611e-03        -317.466970784084         -9.478136e-06
   16          2.714158e-03        -317.466974922057         -4.137973e-06
   17          3.239185e-03        -317.466978643544         -3.721487e-06
   18          2.062657e-03        -317.466980346132         -1.702588e-06
   19          1.590006e-03        -317.466981296735         -9.506032e-07
   20          1.665401e-03        -317.466982115222         -8.184866e-07
   21          1.858412e-03        -317.466982895454         -7.802325e-07
   22          2.336106e-03        -317.466983851458         -9.560039e-07
   23          2.683292e-03        -317.466984764856         -9.133983e-07
   24          2.342737e-03        -317.466985443303         -6.784470e-07
   25          2.034136e-03        -317.466985918055         -4.747512e-07
   26          9.860874e-04        -317.466986091534         -1.734795e-07
   27          3.244632e-04        -317.466986138363         -4.682920e-08
   28          2.851979e-04        -317.466986170998         -3.263415e-08
   29          2.250604e-04        -317.466986193068         -2.207082e-08
   30          2.113313e-04        -317.466986210508         -1.743967e-08
   31          1.405379e-04        -317.466986216833         -6.324626e-09
   32          9.274282e-05        -317.466986221422         -4.589651e-09
---------------------------------------------------------------------------
                      SCF converged in 32 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -1.138094
    1       O            :        0.000000        0.000000        1.919293
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                316.831794669872
 E-N energy              :            (au)               -825.791837138181
 Coulomb energy          :            (au)                184.480223811069
 Exchange energy         :            (au)                 -6.575424703017
 X-C energy              :            (au)                -20.427710374975
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 34.015967513809
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -351.482953735231
 Nuclear energy          :            (au)                 34.015967513809
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.174669862214e+02
                         :      (kcal/mol)             -1.992135399488e+05
                         :        (kJ/mol)             -8.335094516869e+05
                         :            (eV)             -8.638716402734e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -56.177519823721
    1    1    a          :            (au)                -19.171678807472
    2    1    a          :            (au)                 -4.207308176287
    3    1    a          :            (au)                 -2.736688593340
    4    1    a          :            (au)                 -2.736688249345
    5    1    a          :            (au)                 -2.735641668472
    6    1    a          :            (au)                 -0.916936745742
    7    1    a          :            (au)                 -0.392617921000
    8    1    a          :            (au)                 -0.301604984676
    9    1    a          :            (au)                 -0.301601600891
   10    1    a          :            (au)                 -0.287792442162
   11    1    b          :            (au)                -56.175274406133
   12    1    b          :            (au)                -19.159990902808
   13    1    b          :            (au)                 -4.203905224672
   14    1    b          :            (au)                 -2.734902475184
   15    1    b          :            (au)                 -2.734902224479
   16    1    b          :            (au)                 -2.727892162587
   17    1    b          :            (au)                 -0.882024875794
   18    1    b          :            (au)                 -0.325388679152
   19    1    b          :            (au)                 -0.291674670603
   20    1    b          :            (au)                 -0.291670782379
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -179.493705416899
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000029       -0.000091       -2.279692
 Magnitude               :            (au)                        2.279692
                         :         (Debye)                        5.794399
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.559118
 Magnitude               :            (au)                        0.559118
                         :         (Debye)                        1.421136
---------------------------------------------------------------------------
 Total vector            :        0.000029       -0.000091       -1.720574
 Magnitude               :            (au)                        1.720574
                         :         (Debye)                        4.373263
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 32m 23s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
